src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,19 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        driverService.create(new Driver(11L, "Afanasij", "2150120"));
+        manufacturerService.create(new Manufacturer(15L, "Lada", "Russia"));
```

Comment:
Maybe you can test all the Dao methods ;)

Reply:
Yeah)